src/bindings.c

```diff
@@ -506,31 +546,31 @@ umth_input_is_pressed(UmkaStackSlot *p, UmkaStackSlot *r)
 {
 	int keycode = p[0].intVal;
 
-	r->intVal = thg->pressed[keycode];
+	umkaGetResult(p, r)->intVal = thg->pressed[keycode];
 }
 
 void
```

Comment:
I think while migrating, we should also add the comment annotations (#167)

Reply:
I'm adding that as I'm going along.